cmd/client/client.go

```diff
@@ -127,3 +129,18 @@ func sendCmd(cmd string) (string, error) {
 
 	return string(buf[:n]), nil
 }
+
+func usage() {
+
+	// todo: we need a client version?
+	helpList := map[string]string{
```

Comment:
Maybe `helpList` could be a global variable?

Reply:
I agreed, this idea also came into my mind when I was coding. Allow me submit it again.